corehq/apps/sms/api.py

```diff
@@ -301,10 +301,7 @@ def send_message_via_backend(msg, backend=None, orig_phone_number=None):
 
         if backend.domain_is_authorized(msg.domain):
             backend.send(msg, orig_phone_number=orig_phone_number)
-            sms_load_counter("outbound", msg.domain, extra_tags={
-                'status': 'ok',
-                'backend': _get_backend_tag(backend),
-            })()
```

Comment:
I noticed these missing now on datadog https://app.datadoghq.com/dashboard/2z2-9gq-y93/sms?from_ts=1585763526691&fullscreen_section=overview&fullscreen_widget=7463741098486582&live=false&tile_size=m&to_ts=1588355526691&tpl_var_environment=production&fullscreen_start_ts=1585763526691&fullscreen_end_ts=1588355526691&fullscreen_paused=true and traced it back to this. I think both these tags are in use—any particular reason you chose them for removal @snopoke?

Guessing it was an oversight, I'll add you as a reviewer on a PR to revert, but interested to know if there's other context here like this being particularly expensive or something

Reply:
Also it's kind of funny that I was the reviewer here that I missed the connection with a change I'd made only a few days earlier. Making a mental note to do default to commit-by-commit reviews and be more careful. Not that I'll catch everything, but this is the kind of thing I should be looking for in a review—outside context I have that the author might not

Reply:
Were those graphs added recently because they weren't there when I checked

Reply:
https://github.com/dimagi/commcare-hq/pull/27351

Reply:
Yeah I added them the same day I rolled out the new tags, so Apr 22. Seems like it just overlapped with when you started working on this so it slipped through the cracks